src/Model/ValueObject/Web/Url.php

```diff
@@ -18,7 +18,7 @@ class Url
     public function __construct($value)
     {
         $this->guardString($value);
-        $this->guardRegexPattern('/\\Ahttp[s]?:\\/\\//', $value);
+        $this->guardRegexPattern('/\\Ahttp[s]?:\\/\\//', strtolower($value));
```

Comment:
Maybe also convert the `$value` to lowercase?

Reply:
I suspect that could interfere with the querystring/ UrlEncoding.  The question is: if/how we should accept stuff like `HtTps://WwW.pUBLIQ.Be` 
Existing tests https://github.com/cultuurnet/udb3-silex/blob/690cee1c1f6dc58a2d2730c68c708e222240db3f/tests/Organizer/WebsiteNormalizerTest.php#L60 suggest yes.

Reply:
True! We discussed that yesterday and did not want to convert `%C3%A9` to `%c3%a9`